ui-v2/app/templates/components/action-group.hbs

```diff
@@ -1,8 +1,10 @@
-<input type="radio" name="actions" id="actions_{{index}}" checked={{if (eq checked 'checked') 'checked' }} onchange={{action onchange}} value={{index}} />
-<label for="actions_{{index}}">
-    <span>Open</span>
-</label>
-<label for="actions_close">
-    <span>Close</span>
-</label>
-{{yield}}
+{{#if hasBlock }}
```

Comment:
To my understanding this would leave non-block component usage empty given there is no else condition here. Is this right? I would assume that you just don't utilize it for non-block usage, but maybe that throws errors in tests, hence being added here? Anything we could do to avoid empty component rendering causing confusion, or do I have this wrong?

Reply:
Yup that's right, previously, if you used it as a non block component it printed 'Open      Close' with all sorts of returns/whitespace etc, so I just made print nothing if you use it as an inline component. That also kind of made more sense to me - 'You've specified no actions? Ok, you get no menu'

So essentially, the component should never be used as a non block component as you should always specify some actions for inside the dropdown.

Actually, you've just given me an idea. Ideally this would throw an error or similar if you tried to call it inline. No idea right now how you would go about throwing an error from a handlebars template, but happy to look into it? Apart from that I've added some comments at the top of the action-group template file which may help (typically that commit doesn't seem to want to show up here in the PR!). Will merge tomorrow now, cheers Jack!